app/build.gradle

```diff
@@ -120,11 +131,13 @@ android {
     buildTypes {
         release {
             minifyEnabled false
+            resValue("string", "git_hash", getGitHash())
         }
         debug {
             isDefault true
             minifyEnabled false
             enableUnitTestCoverage true
+            resValue("string", "git_hash", getGitHash())
```

Comment:
resources are really useful for things that have different values in different languages/configurations.

This is a constant, so it can be a `buildConfigField` as a constant. Also then we can just declare it once.

Reply:
![image](https://github.com/oxen-io/session-android/assets/160798022/7894eb90-ca07-49f3-8181-bece8f2ebf25)
Done!